core-services/ipi-deprovision/aws.sh

```
@@ -268,49 +270,140 @@ function vpc_has_only_orphaned_eni() {
 
+lb_cleanup_failed=0
+if [[ "${inventory_failed}" -ne 0 ]]; then
+	echo "ERROR: LB inventory incomplete; deferring installer destroy until inventories succeed"
+else
+	# Phase 2: only after all regional inventories succeeded, plan VPC cleanup + destroy.
+	for region in "${regions[@]}"; do
+		echo "deprovisioning in AWS region ${region} ..."
+		elbv2_json="$(cat "${inventory_dir}/elbv2-${region}.json")"
+		classic_json="$(cat "${inventory_dir}/classic-${region}.json")"
+		region_lb_deleted=0
+		aws ec2 describe-vpcs --output json --region ${region} | jq --arg date "${aws_cluster_age_cutoff}" -r '.Vpcs[] | select(.Tags[]? | select(.Key == "expirationDate" and .Value < $date)) | . as $vpc | .Tags[]? | select((.Key | startswith("kubernetes.io/cluster/")) and (.Value == "owned")) | "\($vpc.VpcId) \(.Key)"' > /tmp/clusters
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)